tests/unit/lms/services/canvas_api/_authenticated_test.py

```diff
@@ -94,6 +97,7 @@ def test_get_token(
         basic_client.send.assert_called_once_with(
             "POST",
             "login/oauth2/token",
+            url_stub="",
```

Comment:
The arguments were in the wrong order in this `assert_called_once_with()`. I put them in the right order.